### DISCUSSION TITLE
2024/01/21/install-and-use-of-tinytex-and-xelatex/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#12] [General]
# 2024/01/21/install-and-use-of-tinytex-and-xelatex/

通过R安装了TinyTeX这一LaTeX发行版，并探索了用其编译jupyter notebook的方法。

https://wz.anoms.top/2024/01/21/install-and-use-of-tinytex-and-xelatex/

**Comment by Hazelandinstant** [comment]
这周更新的好早呀！

**Reply by cyclinbox** [reply]
这周东西比较少，所以提早更新了😁

**Comment by EamonNKU** [comment]
谢益辉的博客：https://yihui.org/

**Reply by cyclinbox** [reply]
谢益辉被posit裁员了，呜呜
